src/Styleguide/Pages/Artwork/Sidebar.tsx

```diff
@@ -71,10 +75,29 @@ export class Sidebar extends Component<ArtworkSidebarProps> {
     return (
       <SidebarContainer>
         <Artists artists={artwork.artists} />
+
+        {artwork.is_biddable &&
```

Comment:
Was not sure how much of the checking for auction artwork to do here vs in the sidebar elements. There is no exact rule here, just kinda went with what feels right.

In this case - we wanted to prepend the lot number (very simple string) if the artwork is in the auction and the auction is in progress. Figured that testing for `artwork.is_biddable` should do the trick here.